docs/css/reset.css

```diff
@@ -4,7 +4,7 @@ html;body;div;span;applet;object;iframe;h1;h2;h3;h4;h5,h6,p,blockquote,pre,a,abb
 body, html {
   height: 100%;
   -webkit-font-smoothing: antialiased;
-  font-family: "Lato", "Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif;;
+  font-family: "Helvetica Neue", "Helvetica", Helvetica, Arial, sans-serif;;
```

Comment:
Was this a request by Derek?

Reply:
yes, this was a change Derek wanted made